brave/brave-core
# Adds id to pending contributions table

Resolves https://github.com/brave/brave-browser/issues/7275

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- add pending contribution tip
- make sure that you can delete it as well

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
